tests/mongo-go-driver.yml

```diff
@@ -10,6 +10,7 @@ args:
   - ./x/mongo/driver/topology
 # - ./mongo/integration
   - ./mongo/integration/unified
+  - -count=1
```

Comment:
Please also add `-p=1`, `-parallel=1`, increase timeout above, and remove `-shuffle=on`. We will fix all that later